File: LeetCode/배열, 문자열/2475. Number of Unequal Triplets in Array/2475. Number of Unequal Triplets in Array.py
class Solution:
    def unequalTriplets(self, nums: List[int]) -> int:
        long = len(nums)
        answer = 0

        for i in range(long - 2):
            for j in range(i + 1, long - 1):
                if nums[i] != nums[j]:
                    for k in range(j + 1, long):
                        if nums[k] != nums[i] and nums[k] != nums[j]:
                            answer += 1

        return answer

File: LeetCode/배열, 문자열/2475. Number of Unequal Triplets in Array/2475. Number of Unequal Triplets in Array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var unequalTriplets = function(nums) {
    const long = nums.length;
    let answer = 0;

    for(let i = 0; i < long-2; i++){
        for(let j = i+1; j < long - 1; j++) {
            if (nums[i] !== nums[j]) {
                for (let k = j + 1; k < long; k++) {
                    if(nums[k] !== nums[i] && nums[k] !== nums[j]) answer++;
                }
            }
        }
    }
    return answer;
};